Source/WebCore/Modules/mediastream/gstreamer/GStreamerMediaEndpoint.cpp

```diff
@@ -461,7 +461,12 @@ void GStreamerMediaEndpoint::doSetLocalDescription(const RTCSessionDescription*
 
         GRefPtr<GstWebRTCSCTPTransport> transport;
         g_object_get(m_webrtcBin.get(), "sctp-transport", &transport.outPtr(), nullptr);
-        m_peerConnectionBackend.setLocalDescriptionSucceeded(WTFMove(descriptions), { }, transport ? makeUnique<GStreamerSctpTransportBackend>(WTFMove(transport)) : nullptr);
+
+        std::optional<double> maxMessageSize;
+        if (transport)
+            g_object_get(transport.get(), "max-message-size", &maxMessageSize, nullptr);
```

Comment:
Hmm this won't work I think... Something like this maybe?

```
if (transport) {
    uint64_t maxMessageSizeValue;
    g_object_get(transport.get(), "max-message-size", &maxMessageSizeValue, nullptr);
   maxMessageSize = static_cast<double>(maxMessageSizeValue);
}
```

Reply:
Right, fixed.